requirements-dev.txt

```diff
@@ -0,0 +1 @@
+pytest==6.1.1
```

Comment:
Probably don't need to pin the version down, if something breaks I should be able to respond quickly enough to make sure the build passes again. Personally, I don't really like pinned versions.

Reply:
I changed to get always the latest one ✌️